scripts/ci

```diff
@@ -28,15 +28,15 @@ source scripts/build-deviceos
 
 # Build base and main images
 prepare_base_and_main_image ${BUILDPACK_BASE} \
-    "${BUILDPACK_VERSION}-${BUILDPACK_VARIATION}" \
+    ${BUILDPACK_VERSION} \
     $TAG
 
 
 # Prepare test image
 # FIXME: this uses only the main buildpack variation
 # and doesn't take into account another platform using different buildpack
 prepare_test_image ${BUILDPACK_BASE} \
-    "${BUILDPACK_VERSION}-${BUILDPACK_VARIATION}" \
+    ${BUILDPACK_VERSION} \
```

Comment:
Just as a note: I am not too fond of us losing this feature. We may argue that it was rarely used or it is broken (it isn't currently), but it is very useful to be able to override buildpack for a particular platform and it's also not guaranteed that we are going to keep using just ARM GCC going forward.